eng/pipelines/publish-logs.yml

```diff
@@ -29,4 +29,4 @@ steps:
       targetPath: '$(Build.SourcesDirectory)/artifacts/log/${{ parameters.configuration }}'
       artifactName: '${{ parameters.jobName }} Attempt $(System.JobAttempt) Logs'
     continueOnError: true
-    condition: not(succeeded())
\ No newline at end of file
+    condition: true
```

Comment:
This is just temporary. I want to see the logs to make sure everything is working the way I expect it to. Will remove before merge.